core/src/main/kotlin/behavior/ChatInputCommandBehavior.kt

```diff
@@ -27,9 +30,9 @@ public interface GuildChatInputCommandBehavior : ChatInputCommandBehavior, Guild
 
 
 public interface GlobalChatInputCommandBehavior : ChatInputCommandBehavior,GlobalApplicationCommandBehavior {
-
+     @OptIn(KordUnsafe::class)
      override suspend fun edit(builder: suspend ChatInputModifyBuilder.() -> Unit): GlobalChatInputCommand {
-          val request = ChatInputModifyBuilder().apply { builder() }.toRequest()
+          val request = ChatInputModifyBuilderImpl().apply { builder() }.toRequest()
           val response = service.modifyGlobalApplicationCommand(applicationId,id, request)
```

Comment:
```suggestion
          val response = service.modifyGlobalChatInputApplicationCommand(applicationId, id) { builder() }
```
allows `ChatInputModifyBuilderImpl` to be internal